massa-node/base_config/config.toml

```diff
@@ -221,6 +221,12 @@
     broadcast_slot_execution_traces_channel_capacity = 5000
     # Max slots execution traces to keep in cache
     execution_traces_limit = 320
+    # path to the event cache storage
+    event_cache_path = "storage/event_cache/rocks_db"
+    # maximum number of entries we want to keep in the Event cache
+    event_cache_size = 2000
```

Comment:
I think we can safely make this much bigger (eg. aim for the equivalent of 10 gigabytes)

Reply:
Done